### ISSUE TITLE
Auto update instances with update policy

### ISSUE THREAD

**Issue by ramonsmits** [Particular/ServiceControl#3615]
#### Is your feature related to a problem? Please describe.

Many customer run on outdated unsupported versions of ServiceControl. Primarily this is because these instances requires manual updating in a way that at the moment very cumbersome.

#### Describe the requested feature

Have instances that share the same major or major/minor auto update. There could be a grace period applied to not immediately update to the last compatible version but only after a certain duration has passed.

For example, auto update 7 days after the release or, at minimum 7 days after the release after working hours.

In the user-interface this should also be visible so that an operator sees when an update is scheduled and can either cancel, postpone 

Notifications should be send to the operator email that will state when a certain upgrade will be performed.
